cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaStatusTest.java

```diff
@@ -185,16 +184,18 @@ public void testPauseReconciliationsStatus(VertxTestContext context) throws Pars
 
         Checkpoint async = context.checkpoint();
         kao.reconcile(new Reconciliation("test-trigger", Kafka.RESOURCE_KIND, namespace, clusterName)).onComplete(res -> {
-            assertThat(res.succeeded(), is(true));
-
-            assertThat(kafkaCaptor.getValue(), is(notNullValue()));
-            assertThat(kafkaCaptor.getValue().getStatus(), is(notNullValue()));
-            KafkaStatus status = kafkaCaptor.getValue().getStatus();
-
-            assertThat(status.getConditions().size(), is(1));
-            assertThat(status.getConditions().get(0).getStatus(), is("True"));
-            assertThat(status.getConditions().get(0).getType(), is("ReconciliationPaused"));
-            assertThat(status.getObservedGeneration(), is(1L));
+            context.verify(() -> {
```

Comment:
As you changed this already ... you should be able to use something like this and replace the result success check as well:

```
.onComplete(context.succeeding(pfa -> context.verify(() -> { ... })))
```

Reply:
Updated to use `context.succeeding`.